2024-06-10/README.md

```diff
@@ -0,0 +1,14 @@
+A simple client / server experiment using `requests` and `FastAPI`
+
+## Client Portion
+
+You need to change URL to the host running the server portion. Then, you can attempt to run the functions in challenge.py
+
+## Server portion
+
+To run the server, make sure that you've installed the requirements, and then run
+
```

Comment:
Perhaps we can also include other OS instructions?

Reply:
I'm thinking that's out of scope here. These instructions should do fine on any *nix, and we're not supporting windows systems at this time.